### ISSUE TITLE
Cant see the posts

### ISSUE THREAD

**Issue by Sevada797** [th3unkn0n/osi.ig#12]
Traceback (most recent call last):
  File "main.py", line 20, in <module>
    ig.print_data_()                                                         File "/data/data/com.termux/files/home/osi.ig/.lib/api.py", line 110, in print_data_
    for key, value in collections.Counter(self.tags).most_common():
AttributeError: 'main' object has no attribute 'tags'

<h3>This is the Error code </h3>

**Comment by radryry** [comment]
Found a fix. (I know next to nothing about Linux/termux/hacking ect. as a whole so don't judge) I don't know if it will be the same for you but try "2to3 . -w" and that should rewrite all your py2 script to py3. My other tools were fixed aswell but if you want to fix something specific like I did for trape try "2to3 -w trape" sorry if I'm writing all this wrong but I guess I'm a script kiddy?.. w. E that means.
-cowabunga dudes - follow my IG and say hi @rad.ryry
